frontend/src/metabase/nav/components/AppBar/AppBarToggle.tsx

```diff
@@ -5,18 +5,26 @@ import Tooltip from "metabase/core/components/Tooltip";
 import { SidebarButton, SidebarIcon } from "./AppBarToggle.styled";
 
 export interface AppBarToggleProps {
+  isLogoVisible?: boolean;
   isNavBarOpen?: boolean;
   onToggleClick?: () => void;
 }
 
 const AppBarToggle = ({
+  isLogoVisible = true,
   isNavBarOpen,
   onToggleClick,
 }: AppBarToggleProps): JSX.Element => {
   return (
     <Tooltip tooltip={getSidebarTooltip(isNavBarOpen)}>
-      <SidebarButton onClick={onToggleClick} data-testid="sidebar-toggle">
+      <SidebarButton
+        isLogoVisible={isLogoVisible}
+        onClick={onToggleClick}
+        data-testid="sidebar-toggle"
+        aria-label="sidebar-toggle"
```

Comment:
add `aria-label` so we could use `*byRole` or `cy.button` in Cypress.

Reply:
`aria-label` should be human readable. Something like 
```
t`Toggle sidebar`
```